curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657caf204c0d672a35411c31.md

```diff
@@ -1,15 +1,15 @@
 ---
 id: 657caf204c0d672a35411c31
-title: Step 27
+title: Step 7
 challengeType: 0
-dashedName: step-27
+dashedName: step-7
 ---
 
 # --description--
 
 Each time you roll the dice, you could end up with a `"Three of a kind"`, `"Four of a kind"`, `"Full house"`, `"Straight"` or a random combination of numbers. Based on the outcome, you can make a selection and add points to your score. 
 
-Start by creating an arrow function called `updateRadioOption` that takes `optionNode` and `score` as parameters.
+Start by creating an arrow function called `updateRadioOption` that takes an index and a score value as arguments. It should set the `scoreInputs` at that index to be enabled, set the value of that input to the score, and display `, score = ${score}` in the correct `scoreSpans` element.
```

Comment:
I think we should just say function instead of being strict about an arrow function.

```suggestion
Start by creating a function called `updateRadioOption` that takes an index and a score value as arguments. It should set the `scoreInputs` at that index to be enabled, set the value of that input to the score, and display `, score = ${score}` in the correct `scoreSpans` element.
```

Reply:
Also, in the next step the parameter name used is `optionNode`